src/gui/static/src/app/components/pages/send-skycoin/send-verify/send-verify.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { WalletService } from '../../../../services/wallet.service';
+import { ButtonComponent } from '../../../layout/button/button.component';
+
+@Component({
+  selector: 'app-send-verify',
+  templateUrl: './send-verify.component.html',
+  styleUrls: ['./send-verify.component.scss']
+})
+export class SendVerifyComponent {
+  @ViewChild('sendButton') sendButton: ButtonComponent;
+  @ViewChild('backButton') backButton: ButtonComponent;
+  @Input() transaction: any;
+  @Input() encodedTransaction: string;
+  @Output() onBack = new EventEmitter<boolean>();
+
+  constructor(
+    private walletService: WalletService,
+  ) {}
+
+  send() {
+    if (this.sendButton.isLoading()) {
```

Comment:
This is only necessary if `send()` can be called in some other way than pressing the button.